mappings/com/mojang/blaze3d/vertex/VertexFormatElement.mapping

```diff
@@ -13,10 +13,10 @@ CLASS net/minecraft/unmapped/C_gmhefnuu net/minecraft/client/render/VertexFormat
 		ARG 1 textureIndex
 		ARG 2 dataType
 		ARG 3 type
-		ARG 4 length
+		ARG 4 count
 	METHOD equals (Ljava/lang/Object;)Z
 		ARG 1 o
-	METHOD m_gxxdrrba startDrawing (IJI)V
+	METHOD m_gxxdrrba setupAttribState (IJI)V
```

Comment:
setupAttributeState? I think its important to specify that.

Reply:
`Attrib` is what opengl uses. see https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glGetAttribLocation.xhtml for example

Reply:
alright, perfect then